Gemfile.lock

```diff
@@ -0,0 +1,207 @@
+PATH
```

Comment:
Normally I think it's bad practice to check in a `Gemfile.lock` in a gem repo itself, but in this case, I tried building the gem locally and got tons of version mismatches between the different dependencies because they're all so old (especially some of the test dependencides), and the `.gemspec` was not specific enough -- is that why this is checked in here?

Reply:
I wasted hours trying to quickly update to ruby 2.5/rails 5. When I backed off and built the Gemfile cleanly, there were multiple spec failures due to some incremental update in some dependency. I finally had to go back into a previous CircleCI run that succeeded, and capture the dependency versions, specify them temporarily in the gemspec, then build the Gemfile.lock. I debated checking in the gemspec instead, but didn't love the idea of hardcoding exact versions of all dependencies. I'll make a chore to update this gem - attempting to get the upstream updates from the original, which would include ruby/rails updates.